andela/ah-sealteam-frontend
# #161278870 Users should be able to view and edit their profiles

#### What does this PR do?
- Adds functionality for users to view and update their profiles
#### Description of Task to be completed?
- Users can view their profiles
- Users can update their profiles
#### How should this be manually tested?
   `$ git clone https://github.com/andela/ah-sealteam-frontend.git`
   `$ git checkout feature/Users-should-be-able-to-view-and-edit-their-profiles-#161278870`
   `$ npm install`
   `$ npm start`
   `$ Register account via http://localhost:3000/signup'
- This will redirect you to the profile page `http://localhost:3000/profile`
#### What are the relevant pivotal tracker stories?
[#161278870](https://www.pivotaltracker.com/n/projects/2205273/stories/161278870)
